sites/funmooc/src/backend/base/context_processors.py

```diff
@@ -3,48 +3,23 @@
 """
 import json
 
-from django.conf import settings
-from django.db.models import Q
 from django.utils.translation import get_language_from_request
 
 from cms.models import Page
-from richie.apps.courses.models import Organization
 
 
 def site_metas(request):
     """
     Context processor to add all information required by frontend scripts.
     """
 
-    page = request.current_page or None
-    language = get_language_from_request(request, check_path=True)
-
-    def get_organizations():
-        """
-        Return organization attached to the page and
-        organizations linked to the current page via an organization plugin in any of the
-        placeholders on the page
-        """
-        return list(
-            Organization.objects.filter(
-                Q(
-                    extended_object__organization_plugins__cmsplugin_ptr__language=language,
-                    extended_object__organization_plugins__cmsplugin_ptr__placeholder__page=page,
-                )
-                | Q(
-                    extended_object__exact=page,
-                ),
-                extended_object__title_set__published=True,
-            )
-            .distinct()
-            .values_list("code", flat=True)
-        )
-
     def get_privacy_policy_uri():
         """
         Check if a privacy policy page is published in the request language
         then return its url.
         """
+        language = get_language_from_request(request, check_path=True)
```

Comment:
why not keep it outside of this function?

Reply:
`language` is now used only in one context so I move this variable near to the context which uses it. 🤷‍♂️